README.md

```diff
@@ -91,18 +89,6 @@ If `pa11y.html` demo exists, confirms it is accessible using [Pa11y](http://pa11
 If `package.json` contains a `test` script, confirms it exits with a 0 exit code.
 Runs tests using [Karma](https://karma-runner.github.io) defaulting to Chrome Stable, can be configured to use BrowserStack by using the `--browserstack` flag. You will need the environment variables `BROWSER_STACK_USERNAME` and `BROWSER_STACK_ACCESS_KEY` set. This will run the tests on the minimum version for enhanced experience based on the [FT Browser Support Policy[(https://docs.google.com/document/d/1mByh6sT8zI4XRyPKqWVsC2jUfXHZvhshS5SlHErWjXU).
 
-
-### `init`
-
-Initialises a new component with a full boilerplate folder structure.
-
-This command takes an argument in the form of a component name, which will populate all of the relevant files within that tree. Defaults to `o-component-boilerplate`.
-
-e.g.
-```
-obt init o-my-new-component
-```
-
 ## Migration guide
```

Comment:
Need to mention that --watch and --run-server have been removed and that obt demo has been replaced with obt dev for development workflows